app/assets/stylesheets/helpers/_core.scss

```diff
@@ -298,38 +298,6 @@ h4 {
   }
 }
 
-#wrapper.businesslink #legacy-sources,
-#wrapper.directgov #legacy-sources {
-  display: block;
-}
-
-#wrapper.businesslink #legacy-sources li.businesslink {
-	background-image: image-url("businesslink-logo.png");
```

Comment:
I wouldn't worry about prototyping, but can't speak for performanceplatform-selfserve. Maybe ask @jonnywyatt about that? I assume it's there to mimic what's on GOV.UK?